server-timing/class-perflab-server-timing.php

```diff
@@ -227,16 +227,11 @@ public function on_template_include( $passthrough = null ) {
 			return $passthrough;
 		}
 
-		ob_start();
-		add_action(
-			'shutdown',
-			function() {
-				$output = ob_get_clean();
+		ob_start(
+			function( $output ) {
 				$this->send_header();
-				echo $output;
-			},
-			// phpcs:ignore PHPCompatibility.Constants.NewConstants
-			defined( 'PHP_INT_MIN' ) ? PHP_INT_MIN : -1000
```

Comment:
Curious why this action was configured to run early. Wouldn't it be better to run as late as possible?